src/wallet3/db_schema.cpp

```diff
@@ -47,11 +49,15 @@ namespace wallet
             last_scan_height INTEGER NOT NULL DEFAULT -1,
             scan_target_hash TEXT NOT NULL,
             scan_target_height INTEGER NOT NULL DEFAULT 0,
-            output_count INTEGER NOT NULL DEFAULT 0
+            output_count INTEGER NOT NULL DEFAULT 0,
+            spend_priv BLOB,
+            spend_pub BLOB,
+            view_priv BLOB,
+            view_pub BLOB
           );
 
           -- insert metadata row as default
-          INSERT INTO metadata VALUES (0,0,"testnet",0,0,-1,"",0,0);
+          INSERT INTO metadata VALUES (0,0,"testnet",0,0,-1,"",0,0,"","","","");
```

Comment:
SQL string literals are single-quoted `'abc'`, not `"abc"` (double quoting in SQL is for identifier quoting, e.g. `INSERT INTO "metadata" VALUES ();`).  Double-quoted strings is a backward-compatibility hack for compatibility with MySQL, but SQLite is moving away from it: the latest release no longer allows their use by default in the CLI.